wheel_of_jeopardy/wheel_of_jeopardy/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+
+class Category(models.Model):
+    '''
+    Stores a category entry. One-to-many relationship with Question.
+    '''
+    category_title = models.CharField(max_length=200)
+
+
+class Question(models.Model):
+    '''
+    Stores a single question and ties it to a category. Many-to-one relationship with Category.
+
+    GameSession used as optional secondary foreign key, allows a list of already asked questions to be maintained
+    during the game.
+    '''
+    question_text = models.CharField(max_length=200)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    game_session = models.ForeignKey('GameSession', null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
Should also include a point total associated with the question? and should include an Answer model as well

Reply:
I somehow missed this comment when I merged but yes agreed, I'll update now